mlir/include/mlir/Dialect/MIOpen/MIOpenOps.td

```diff
@@ -769,8 +770,8 @@ def MIOpen_XdlopsGemmV2Op:
                    MemRefOf<[F32, F16, BF16, I8]>:$matrixB,
                    IndexAttr:$ldsBufferOffsetA,
                    IndexAttr:$ldsBufferOffsetB,
-                   Index:$regOffsetA,
-                   Index:$regOffsetB,
+                   IndexAttr:$regOffsetA,
+                   IndexAttr:$regOffsetB,
```

Comment:
Thanks!

Reply:
No problem. If I don't do this, we'd have to generate redundant IRs to calculate the right offset in this PR. Now looking forward to get rid of those in upcoming PRs...